scripts/runtests.py

```diff
@@ -44,11 +46,21 @@ def _get_strict_params(stub_path: str) -> list[str]:
 
 
 def main() -> None:
-    try:
-        path = sys.argv[1]
-    except IndexError:
-        print("Missing path argument in format <folder>/<stub>", file=sys.stderr)
-        sys.exit(1)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--run-stubtest",
+        type=str.lower,
+        choices=_RUN_STUBTEST_CHOICES,
+        default=None,
+        help="Run or skip stubtest without prompt. Only use this if you trust the package you are testing.",
+    )
+    parser.add_argument(
+        "path", type=str, help="Path of the stub to test in format <folder>/<stub>, from teh root of the project."
+    )
+    args = parser.parse_args()
+    path: str = args.path
+    run_stubtest: str | None = args.run_stubtest
+
     assert os.path.exists(path), rf"Path {path} does not exist."
     path_tokens = Path(path).parts
     assert len(path_tokens) == 2, "Path argument should be in format <folder>/<stub>."
```

Comment:
We can have some nicer error messages here if we're now using `argparse` (same for the `assert folder in {"stdlib, "stubs"}` line just below, but it's too far out of range for me to make a suggestion :)

```suggestion
    if not os.path.exists(path):
        parser.error(rf"Path {path} does not exist.")
    path_tokens = Path(path).parts
    if len(path_tokens) != 2:
        parser.error("Path argument should be in format <folder>/<stub>.")
```

Reply:
Oh sweet! I've re-ordered them a bit too to better guide a new contributor through the more obvious errors.